tests/e2e/hcp_cluster_test.go

```diff
@@ -324,4 +328,111 @@ var _ = Describe("HCP cluster testing",
 				Expect(clusterDesc.OperatorIAMRoles).To(HaveEach(MatchRegexp("arn:aws:iam::[0-9]{12}:role/.+")))
 				Expect(clusterDesc.State).To(Equal(constants.Ready))
 			})
+
+		It("ROSA CLI cluster creation should show install/uninstall logs - [id:75534]", labels.Critical, labels.Runtime.Day2,
+			func() {
+				var (
+					ocmResourceService = rosaClient.OCMResource
+					hostedCP           = true
+					caseNumber         = "75534"
+					region             = clusterConfig.Region
+					clusterName        = common.GenerateRandomName("ocp-75534", 2)
+					installerRoleArn   string
+				)
+				By("Create a HCP cluster")
+				By("Prepare a vpc for the testing")
+				vpc, err := vpc_client.PrepareVPC(caseNumber, region, "", false, "")
+				Expect(err).ToNot(HaveOccurred())
+				defer vpc.DeleteVPCChain()
+
+				subnetMap, err := profilehandler.PrepareSubnets(vpc, region, []string{}, false)
+				Expect(err).ToNot(HaveOccurred())
+
+				subnetsFlagValue := strings.Join(append(subnetMap["private"], subnetMap["public"]...), ",")
+				rosaclient := rosacli.NewClient()
+
+				By("Create account-roles of hosted-cp")
+				_, err = ocmResourceService.CreateAccountRole("--mode", "auto",
+					"--prefix", "akanni",
+					"--hosted-cp",
+					"-y")
+				Expect(err).To(BeNil())
+
+				By("Get the installer role arn")
+				accountRoleList, _, err := ocmResourceService.ListAccountRole()
+				Expect(err).To(BeNil())
+				installerRole := accountRoleList.InstallerRole("akanni", hostedCP)
+				Expect(installerRole).ToNot(BeNil())
+				installerRoleArn = installerRole.RoleArn
+
+				By("Create managed=false oidc config in auto mode")
+				output, err := ocmResourceService.CreateOIDCConfig("--mode", "auto",
+					"--prefix", "akanni",
+					"--managed=false",
+					"--installer-role-arn", installerRoleArn,
+					"-y")
+				Expect(err).To(BeNil())
+				oidcPrivodeARNFromOutputMessage := common.ExtractOIDCProviderARN(output.String())
+				oidcPrivodeIDFromOutputMessage := common.ExtractOIDCProviderIDFromARN(oidcPrivodeARNFromOutputMessage)
+				unmanagedOIDCConfigID, err := ocmResourceService.GetOIDCIdFromList(oidcPrivodeIDFromOutputMessage)
+				Expect(err).To(BeNil())
+				textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+				Expect(textData).To(ContainSubstring("Created OIDC provider with ARN"))
+				output, err, _ = rosaclient.Cluster.Create(
+					clusterName,
+					"--region", region,
+					"--replicas", "3",
+					"--subnet-ids", subnetsFlagValue,
+					"--hosted-cp",
+					"--oidc-config-id", unmanagedOIDCConfigID,
+					"--billing-account", profile.ClusterConfig.BillingAccount,
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Create operator roles")
+				output, err = ocmResourceService.CreateOperatorRoles(
+					"-c", clusterName,
+					"--mode", "auto",
+					"-y",
+				)
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
Why not using the 
```
flags, err := profilehandler.GenerateClusterCreateFlags(customProfile, rosaClient)
				Expect(err).ToNot(HaveOccurred())

				command := "rosa create cluster --cluster-name " + testingClusterName + " " + strings.Join(flags, " ")
				rosalCommand := config.GenerateCommand(command)
```

See https://github.com/openshift/rosa/blob/master/tests/e2e/test_rosacli_cluster.go#L3423 as an example

Reply:
@radtriste i was told by Xue to not use `profilehandler.GenerateClusterCreateFlags` because its produces resource leakage while working on another automation task where i have created a HCP cluster.
https://github.com/openshift/rosa/pull/2177/commits/baa18002913bd4de184a6ee81371bd868cdbaa19